kano_settings/config_file.py

```diff
@@ -108,7 +108,7 @@ def set_setting(variable, value):
     if username == 'root':
         return
 
-    logger.debug('config_file / set_setting: {} {}'.format(variable, value))
+    logger.debug("config_file / set_setting: {} {}".format(variable, value.encode('utf-8','ignore')))
```

Comment:
It seems like this `encode` call here will cause an exception when `value` is not a string, which is possible.